gradle.properties

```diff
@@ -1,6 +1,7 @@
 org.gradle.caching=true
 group=org.ballerinalang
 version=swan-lake-preview8-SNAPSHOT
+ballerinaNextSnapshotVersion=2.0.0-Preview8-SNAPSHOT
```

Comment:
lets remove this constant and use `ballerinaVersion`

Reply:
Done.